src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationKeyVaultOptions.cs

```diff
@@ -14,9 +14,13 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
     /// </summary>
     public class AzureAppConfigurationKeyVaultOptions
     {
+        private static readonly TimeSpan DefaultRefreshInterval = TimeSpan.FromHours(12);
+        private static readonly TimeSpan MinimumRefreshInterval = TimeSpan.FromHours(1);
```

Comment:
I would suggest we don't introduce a default or minimum interval for secret refresh. This value can be very different for each user depending on their policy requirements. Leave it to users.

Reply:
Removed `MinimumRefreshInterval`